api-server/src/main/java/io/enmasse/api/v1/http/HttpAddressServiceBase.java

```diff
@@ -282,6 +284,33 @@ protected Response internalDeleteAddresses(SecurityContext securityContext, Stri
         });
     }
 
+    protected Response patchInternal(@Context SecurityContext securityContext, String namespace, String addressSpace, String addressName,
+                                     CheckedFunction<JsonNode, JsonNode, JsonPatchException> patcher) throws Exception {
+        return doRequest("Error patching address space " + addressName, () -> {
+            verifyAuthorized(securityContext, namespace, ResourceVerb.patch);
+
+            Optional<Address> existing = apiHelper.getAddress(namespace, addressSpace, addressName);
+
+            if (!existing.isPresent()) {
+                return Response.status(404).entity(Status.notFound("Address", addressName)).build();
+            }
+            Address existingAddress = existing.get();
+
+            ObjectMapper mapper = new ObjectMapper();
```

Comment:
Nitpick ,but maybe make this static final and just reuse it?

Reply:
Fixed.